types/common/common.go

```diff
@@ -293,7 +293,7 @@ func BytesToAddress(b []byte) Address {
 }
 
 func HexStringToAddress(s string) Address {
```

Comment:
Let's divide this method into safe and unsafe: `HexStringToAddress` (that would return error) and `MustHexStringToAddress` (that would panic)
And then only call unsafe from tests or if argument is static